optuna/dashboard.py

```diff
@@ -252,7 +252,7 @@ def _get_this_source_path():
     return path
 
 
-def serve(study, bokeh_allow_websocket_origins=None):
+def _serve(study, bokeh_allow_websocket_origins=None):
```

Comment:
A minor nitpick but how about making `bokeh_allow_websocket_origins` mandatory to make the type hints simpler. I also believe there's a (rather small) performance gain since the default value does not have to be maintained by the function.

Reply:
That makes sense. `_serve` is only called by CLI, and your suggestion change rarely affects users. But, I'd like to create a new PR for the change because it is not directly related to this PR. What do you think of it?

Reply:
Sounds good to me.